base/int.jl

```diff
@@ -198,8 +205,11 @@ See also: [`signed`](@ref), [`sign`](@ref), [`signbit`](@ref).
 julia> unsigned(-2)
 0xfffffffffffffffe
 
-julia> unsigned(2)
-0x0000000000000002
+julia> unsigned(Int8(2))
```

Comment:
Why introduce `Int8` here which is very rarely used in practice? Most people will apply it to a standard integer type, or?

Reply:
Because `Int8` is a much easier type to reason about since it is short?

Reply:
Yes. Also, to point out that the size of the unsigned type matches, invisible if all examples are 64 bits.